docs/configuration-validation.md

```diff
@@ -27,3 +27,9 @@ Add `configValidationMode` to your service configuration with one of the followi
 | error | Fail invoked command and output configuration errors. |
 | warn  | Output configuration errors in form of warnings.      |
 | off   | Suppress configuration errors.                        |
+
+_e.g. in serverless.yml:_
+```
+service: my-project
```

Comment:
Let's remove `service` property, it's not relevant here (and our goal is not to present here the fully working service configuration, for which even `service` will not be good enough)